docs/reference/setup/sysconfig/virtual-memory.asciidoc

```diff
@@ -18,3 +18,10 @@ To set this value permanently, update the `vm.max_map_count` setting in
 
 The RPM and Debian packages will configure this setting automatically. No
 further configuration is required.
+
+You can find out the current mmap count of a running Elasticsearch process using the following command:
```

Comment:
I reckon it's more usable not to split this sentence either side of the displayed example:

```suggestion
You can find out the current mmap count of a running Elasticsearch process
using the following command, where `$PID` is the process ID of the running
Elasticsearch process:
```